backend/backends/pytorch_backend.py

```
@@ -21,6 +21,7 @@
 )
 from ..utils.cache import get_cache_key, get_cached_voice_prompt, cache_voice_prompt
 from ..utils.audio import load_audio
+from ..utils.whisper_stt import transcribe_whisper_pytorch
```

Ruff (0.15.20)

[error] Module level import not at top of file

(E402)